src/Graphics/Effect/EffectAnnotation.cs

```diff
@@ -58,7 +58,8 @@ public EffectParameterType ParameterType
 		#region Private Variables
 
 		IntPtr values;
-
+		internal string? cachedString 
```

Comment:
May as well just initialize this, to skip the Nullable container:

```suggestion
		internal string cachedString = string.Empty;
```